src/Audit/UntrustedRoles.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drutiny\Plugin\Drupal8\Audit;
+
+use Drutiny\Audit;
+use Drutiny\Sandbox\Sandbox;
+
+/**
+ * User #1
+ */
+class UntrustedRoles extends Audit {
+
+  /**
+   *
+   */
+  public function audit(Sandbox $sandbox) {
+    // Get the machine names of the untrusted roles
+    $untrusted_roles = $sandbox->getParameter('untrusted_roles', ['anonymous', 'authenticated']);
+    
+    // Load all of Drupal's permissions so that we have access to the
+    // "restrict access" property.
+    $command = "echo json_encode(\\Drupal::service('user.permissions')->getPermissions());";
+    $command = "\"$command\"";
+    $all_permissions = $sandbox->drush(['format' => 'json'])->ev($command);
```

Comment:
Another way to write this:

```php
$all_permissions = $sandbox->drush()->evaluate(function () {
 return \\Drupal::service('user.permissions')->getPermissions();
});
```

Its still 3 lines of code but much cleaner. The anonymous function is based into Drupal's runtime environment and the `evaluate` call handles the transfer of the returned output back into the current php runtime.

Reply:
Oh this is great! That will come in handy later when needing to leverage other Drupal functions. Thanks for the protip.